proto/apidocs.swagger.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "proto/v1beta1/stencil.proto",
+    "title": "gotocompany/stencil/v1beta1/stencil.proto",
```

Comment:
Can you please check if this change is required ?

Reply:
reverted to proto.It will be removed after migration